README.md

```diff
@@ -12,3 +12,17 @@ Objectives
 
 
 Project will be a login Page allowing admins to increment a counter and users to view the count.
+
```

Comment:
Brief description, why should I read these, so maintainer knows why they are there

Reply:
This is my dump of links that might save me.... 👎